com.trollworks.gcs/src/com/trollworks/gcs/ui/widget/dock/DockContainer.java

```diff
@@ -332,4 +344,72 @@ public void layoutContainer(Container parent) {
             current.setBounds(insets.left, insets.top + height, width, remaining);
         }
     }
+
+    private void setupDockablesSwitchKeys() {
+        // Standard modifier for shortcuts _should_ be CTRL on win and linux and CMD on mac
+        int       standardModifier = Toolkit.getDefaultToolkit().getMenuShortcutKeyMaskEx();
```

Comment:
Don't want to use this on Mac, since Cmd-Tab and Shift-Cmd-Tab is already a system-wide key binding (it is for switching apps via the keyboard). Use of Ctrl on the Mac is sufficient.